src/components/EditOnGithub.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { documentPathToGithubMarkdown } from 'helpers/docpath';
+
+export default function({ pagepath }) {
+  try {
+    const url = documentPathToGithubMarkdown(pagepath);
+    return (
+      <span className="editongithub">
+        <a className="editongithub__link" href={url} target="_blank" >
+          Edit on github
```

Comment:
GitHub.

Can you also add a FontAwesome icon? `<i class="fa fa-pencil" aria-hidden="true"></i>`